### PR TITLE
Feature/hu2004 create shopping cart

### PR DESCRIPTION
**Pull request by ana03ap**
desktop image
image

mobile image
image

I worked on the shopping cart page of an online store where I added three products. Then, I implemented a summary section that displayed the checkout information such as Subtotal, Discounts, etc. I made sure that the layout was organized and user-friendly. Also, I applied appropriate styles to match the overall design of the site, making sure that the page was responsive and adapted well to different devices and screen sizes.

### CODE REVIEW

**Review by gdgalvis — APPROVED** [review]
All good so far

**Review by Jcoll05 — APPROVED** [review]
Good.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (tllach) [inline .vscode/settings.json:3]: This file doesn't need to be here; it is just for your local deployment. I recommend deleting it

**Review by tllach — APPROVED** [review]
Seems okay

**Pull request closed without merging**